pkg/deviceplugin/api.go

```diff
@@ -70,13 +72,14 @@ func NewDeviceInfo(state string, nodes []pluginapi.DeviceSpec, mounts []pluginap
 
 // NewDeviceInfoWithTopologyHints makes DeviceInfo struct with topology information provided to it.
 func NewDeviceInfoWithTopologyHints(state string, nodes []pluginapi.DeviceSpec, mounts []pluginapi.Mount, envs map[string]string,
-	topology *pluginapi.TopologyInfo) DeviceInfo {
+	annotations map[string]string, topology *pluginapi.TopologyInfo) DeviceInfo {
```

Comment:
Hello.!

What I say would be just a nitpicking, but can I ask if there is a specific reason why 'annotations' is here is before 'topology', and at the lines 81-82 opposite.?

Reply:
done. Thank you for the review!